src/edit/handles/FreeRotateHandle.js

```diff
@@ -1,44 +1,48 @@
 L.FreeRotateHandle = L.EditHandle.extend({
   options: {
-    TYPE: 'freeRotate',
+    TYPE: "freeRotate",
```

Comment:
We prefer single strings where possible

```suggestion
    TYPE: 'freeRotate',
```

Reply:
got it, frankly that was visual studio code auto formatting. should have double checked.

Reply:
yeah i mean, one way isn't better than the other, it's just preference. The essential thing is to maintain consistency.